src/main/java/com/example/WorldOfAnimals/services/AnimalsService.java

```diff
@@ -0,0 +1,34 @@
+package com.example.WorldOfAnimals.services;
+
+
+import com.example.WorldOfAnimals.models.AnimalBehavior;
+import com.example.WorldOfAnimals.models.Animals;
+import com.example.WorldOfAnimals.models.AnimalsGroup;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+
+@Service
+public class AnimalsService {
+
+    private final Animals[] animals = Animals.values();
```

Comment:
This array need delete because Animals have static method

Reply:
done